src/amo/components/LanguageTools/styles.scss

```diff
@@ -43,4 +43,8 @@
   td {
     padding: 6px;
   }
+
+  td {
+    min-height: 31px;
```

Comment:
Could you use line-height or reference a variable relating to font size maybe? It seems a bit fragile.

Reply:
`line-height` did not work, I guess that's because there is no content. I'll try to come up with a variable.